Index about html

Benmuiruri opened the pull request:
### Website for an upcoming F1 Sustainability Summit.
**Website features include:**

- The home page has a link in the menu to the about page.
- The logo in the header links to the home page.
- At 768px wide, the Navbar is hidden and replaced by a hamburger menu
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- The mobile menu has a close (X) button that closes the menu.
- The section "Featured speakers" is created dynamically in JavaScript.
- The details of the Featured Speakers are retrieved from an array of objects 

### Project Video Description 

- You can view a video describing the functionality of the website on [loom](https://www.loom.com/share/1bff608d8ca44673b79ef04b165a6e3c)

Benmuiruri committed:
Add html, css and js lint configs

Benmuiruri committed:
Add index and about page head section

Benmuiruri committed:
Add .gitignore file

Benmuiruri committed:
Add fonts

Benmuiruri committed:
Add top and nav html

Benmuiruri committed:
Add top and nav css styles

Benmuiruri committed:
Add mobile modal and hero html content

Benmuiruri committed:
Add hero section styles for mobile and web view

Benmuiruri committed:
Update nav css and add root font

Benmuiruri committed:
Update nav links spacing

Benmuiruri committed:
Add main program section mobile and web html and css

Benmuiruri committed:
Add partner section web and mobile html and css

Benmuiruri committed:
Add web and mobile footer html and css

Benmuiruri committed:
Add web and mobile about hero section html and css

Benmuiruri committed:
Add Attributioon section in about page

Benmuiruri committed:
Change rem units to em units

Benmuiruri committed:
Update attribution section web and mobile css

Benmuiruri committed:
Add past event html and css

Benmuiruri committed:
Add title background image

Benmuiruri committed:
Remove footer out of main

Benmuiruri committed:
Add margin and padding to events images

Benmuiruri committed:
Update about page hero vh

Benmuiruri committed:
Add on hover effect on event image

Benmuiruri committed:
Add hero title background image

Benmuiruri committed:
Add mobile modal in styles css, create speaker list in main.js

Benmuiruri committed:
Add speakers section from main.js

Benmuiruri committed:
Add speaker section CSS styles

Benmuiruri committed:
Disable unused nav links

Benmuiruri committed:
Style mobile menu modal

Benmuiruri committed:
Fix js linter errors

Benmuiruri committed:
Fix CSS lint errors

ShoiraTa commented:
Hi  👋🏻 ,
While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😢 

### Invalid Code Review Request

- Video presentation is missing
- Readme is not updated

 **Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray** 

Cheers and Happy coding! 👏🏻👏🏻 

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear._

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

***

_Invalid Code Review Request Does not count into the code reviews limit._

Benmuiruri commented:
Hi @ShoiraTa thanks for the feedback. I didn't read the part about adding the video to the PR description. 

Does this invalid code Review request count as 1 of 3. Or do I still have the 3 reviews left? Thanks

ShoiraTa commented:
You still have 3 reviews 👍🏻

Benmuiruri committed:
Update project README.md

Benmuiruri committed:
Add video description in README.md

Donard97 commented on about.html:
```diff
@@ -0,0 +1,199 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About F1 Sustainability Strategy</title>
+    <link rel="stylesheet" href="./css/reset.css" />
```
- [OPTIONAL] Well done on personalizing the title of your project but I suggest you rename it for example only F1 Sustainability Strategy to make it perfect. Keep rocking 💪

Donard97 commented on about.html:
```diff
@@ -0,0 +1,199 @@
+        </ul>
+      </section>
+      <!-- Desktop Nav-->
+      <nav class="desktop-nav">
+        <ul class="inner-content">
+          <li>
+            <a href="index.html" class="nav-logo"
```
- [x] Well done on adding the menu for desktop screens but kindly move them more to the left following the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

Here you have a screenshot of your navbar:

![image](https://user-images.githubusercontent.com/74506933/157509609-08ceaf73-940e-4b78-a360-37d2d11695bc.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/157509659-e2eaaed5-bed6-4d6b-8c11-398d7e05ef91.png)

Donard97 commented on index.html:
```diff
@@ -0,0 +1,199 @@
+          <li><a href="#" class="disabled-link highlighted">F1 Sustainability</a></li>
+        </ul>
+      </nav>
+      <!-- Mobile Nav -->
```
- [x] You did an amazing job adding the mobile nav and styling it but please make the background of the header the same as the section following the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/157508218-f6e5423a-a1b4-4206-b076-6cf4a81d1f3f.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/157508374-bf090234-23c4-4a32-b135-291faf7c97a1.png)

Donard97 commented on index.html:
```diff
@@ -0,0 +1,199 @@
+      </div>
+    </header>
+    <main>
+      <section class="hero" id="hero-content">
```
- [ ] You did an amazing job adding items and styling the first section for the desktop version but kindly decrease a little the size of the fonts or raise the height of the section to match your style with the style from the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Keep rocking 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/157509324-9d725c7e-9d54-439f-8607-3575a12230ed.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/157509163-f8e10151-b49b-4441-8dd8-255d23dfa72d.png)

Donard97 commented on about.html:
```diff
@@ -0,0 +1,199 @@
+              <img src="./images/mercedes-engine.jpg" alt="" />
+              <p class="top-centered">2021</p>
+              <p class="centered">F1 Sustainability Summit 2021, Paris</p>
+            </div>
```
- [x]  I suggest you decrease the font of the text because it looks like it is overwriting each other. Keep rocking 💪

Donard97 commented on about.html:
```diff
@@ -0,0 +1,199 @@
+    </main>
+    <footer class="footer">
+      <div class="inner-content">
+        <img src="./images/cc-license.png" class="footer-img" alt="" />
```
- [x] Well done on adding the footer and styling it but kindly change the background of the footer for desktop version following the design.

Here you have a screenshot of your about footer:

![image](https://user-images.githubusercontent.com/74506933/157511609-ee9b3c9d-452b-4769-bfdb-53ebeeb03ee8.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/157511705-3725dd08-9af5-468a-8be8-9a5b50e85617.png)

Donard97 commented on about.html:
```diff
@@ -0,0 +1,199 @@
+          <h2>Partners</h2>
+          <hr class="orange-line" />
+        </div>
+        <div class="partners-wrapper inner-content">
+          <img src="./images/formula-1-logo.png" class="partner-logo" alt="" />
+          <img src="./images/kakao.png" class="partner-logo" alt="" />
+          <img src="./images/naver.png" class="partner-logo" alt="" />
+          <img src="./images/google.png" class="partner-logo" alt="" />
+          <img src="./images/airbnb.png" class="partner-logo" alt="" />
```
- [x] I suggest you add alt attributes and not leave them blank. Kepp coding 💪

Donard97 commented on index.html:
```diff
@@ -0,0 +1,199 @@
+    <main>
+      <section class="hero" id="hero-content">
+        <div class="hero-inner-content inner-content">
+          <h1>"Hello, Formula One Fans"</h1>
+          <div class="greeting">
+            <p class="large-text">
+              Formula One <br />Global Summit <br />
+              <span>2022</span>
+            </p>
+          </div>
```
- [x] Kindly change the paragraph to h1 because it is the biggest text on the page and the h1 to h2 or h3. Keep shining 💪

Donard97 commented on js/main.js:
```diff
@@ -0,0 +1,164 @@
+// List of speaker objects
+
+const speaker1 = {
+  imageSrc:
+    'https://res.cloudinary.com/optprime/image/upload/v1646837183/module1-capstone/ross-brawn_qgmohf.jpg',
+  speakerName: 'Ross Brawn',
```
- Well done adding the section "Featured speakers" dynamically with JavaScript. Keep this flow 👏

Benmuiruri replied:
Thanks 🙏

Donard97 commented on js/main.js:
```diff
@@ -0,0 +1,164 @@
+const speakerSection = document.querySelector('#speakers');
+// Create inner Div
+
+const innerDiv = document.createElement('div');
+innerDiv.className = 'inner-content';
+const speakerHeader = document.createElement('div');
+speakerHeader.className = 'about-section-header';
+const speakerTitle = document.createElement('h2');
```
- Good job on building the section from Javascript. 👏

Benmuiruri replied:
Thanks 🙏

Donard97 commented on index.html:
```diff
@@ -0,0 +1,199 @@
+          <h2>Partners</h2>
+          <hr class="orange-line" />
+        </div>
+        <div class="partners-wrapper inner-content">
+          <img src="./images/formula-1-logo.png" class="partner-logo" alt="" />
+          <img src="./images/kakao.png" class="partner-logo" alt="" />
+          <img src="./images/naver.png" class="partner-logo" alt="" />
+          <img src="./images/google.png" class="partner-logo" alt="" />
+          <img src="./images/airbnb.png" class="partner-logo" alt="" />
+        </div>
```
- [x] Kindly add alt attributes to images and please do not leave them blank. Keep coding 💪

Donard97 commented on README.md:
```diff
@@ -0,0 +1,52 @@
+## Show your support
+
+Give a ⭐️ if you like this project!
+## Video Presentation
+
```
- Well done on the presentation, I really like it. You were too specific and too clear. You manage to talk about everything that you used and we saw responsiveness on both screens. What I would suggest you in the next presentation is to show us a piece of code that you find interesting or you are proud of. Otherwise, it was amazing 👏 💪

Benmuiruri replied:
Thanks, I'll write a script before taking the video to practice what I will say within the 5 minutes and ensure I include a bit of code. Thanks for the feedback

Donard97 reviewed (changes requested):
## Hello @Benmuiruri , 👋  

_You did an amazing job so far! 👏 💯_ 

**_You did most of the work. You have a few more steps to pass but you are already there! 💪 👍_** 

![hgiphy](https://user-images.githubusercontent.com/74506933/157517152-3ec69857-433c-4624-8d36-4d6427966d91.gif)

## Highlights 🏁 ✅ 

- Presentation was amazing ✔️ 
- The Readme file is professional ✔️ 
- Gitflow is used in the correct way ✔️ 
- Code is clean ✔️ 
- There are a lot of commits and they are meaningful ✔️ 
- PR is descriptive ✔️ 

## Required Changes ♻️

- [ ] Kindly fix the live link because it is showing a blank page.
- [ ] Please check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

Benmuiruri committed:
Update nav links margin left value

Benmuiruri committed:
Update about page title

Benmuiruri committed:
Update background position of greeting div image

Benmuiruri committed:
Increase hero section height

Benmuiruri committed:
Update event image overlay text size

Benmuiruri committed:
Update p to h1

Benmuiruri committed:
Add image alt in html images and javascript-added images

Benmuiruri committed:
Change about page footer background color

Benmuiruri committed:
Change footer element alignment

Benmuiruri commented:
Hi @Donard97 thanks for this wonderful review. It's quite detailed which makes it easy to make the changes. I made the following changes 

- Reduced the `margin-left` on the nav links to position them similar to the template 
- Updated the about page title 
- Changed the `background-position` of the image to make it similar to the template.
- Increased the hero section viewport height
- Decreased size of event image overlay text
- Changed the `p `to a `h1`
- Added image `alt` in `HTML `and for the `Javascript `loaded speaker images
- Changed about page footer `background-color`

PS - The live link is blank because it is deploying from `main `branch and the branch currently does not have any `html`. Once I merge this PR, all the good stuff will be available for everyone to see 😃

prolajumokeoni commented on about.html:
```diff
@@ -0,0 +1,199 @@
+    <main>
+      <section class="hero" id="hero-content">
+        <div class="hero-inner-content inner-content about-inner-content">
+          <h2>Hello, Formula One Fans</h2>
```
- [x] The top background image is still missing in the hero-content section, please add the background image just as the previous reviewer requested, this makes your design look more like the design you are following 👍🏿 

## OPTIONAL

- [x] Kindly add the right font to this header on the mobile screen 👍🏿

prolajumokeoni commented on index.html:
```diff
@@ -0,0 +1,200 @@
+            </p>
+          </div>
+        </div>
+      </section>
```
- [x] Kindly add more padding in this section so it does not overlap with the program section, currently, there is an overlap, please add more margin just as the previous reviewer also requested, this is what I see on smaller desktop screen size (992px)

- Currently, this is what I see ⬇️ 

![image](https://user-images.githubusercontent.com/69638013/157594266-bc4a897b-387f-432f-a749-52478765e68d.png)

prolajumokeoni reviewed (changes requested):
Hi @Benmuiruri 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
✔️ No linter errors

✔️ Github flow was correctly used

✔️ Beautiful presentation
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
**_Check comments under review_**

![almost-gif](https://user-images.githubusercontent.com/69638013/157602946-329ed994-8b7f-4fbb-9d37-81dfed2fdf77.png)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Benmuiruri committed:
Change hero h2 font family

Benmuiruri committed:
Add margin-bottom to hero section

Benmuiruri committed:
Add position absolute to mobile-nav

Benmuiruri committed:
Add position absolute to desktop-nav

Benmuiruri commented:
HI, @prolajumokeoni Thanks for the review. I have made the changes you requested 

- Changed the hero h2 font-family
- Added some margin-bottom to the hero section
- Added position absolute to mobile-nav and desktop-nav to get the hero section background to apply to the nav as well.

ShoiraTa commented on about.html:
```diff
@@ -0,0 +1,199 @@
+          <h2>Formula 1 Sustainability 2022 Logo</h2>
+          <hr class="orange-line" />
+        </div>
+        <p class="web-attribution-text">
+          The logo of Formula 1 Sustainability 2022 summit was decided through
+          the logo competition from 8 June to 7 July. After online public voting
+          and Selection Committee's judging process, the logo from Naresh
+          Agrawal won the prize. After consultation with Mr. Agrawal we
+          optimized the prize-winning-logo for this year's summit as following:
+        </p>
+        <p class="mobile-attribution-text">
+          The logo of Formula 1 Sustainability 2022 was decided through the logo
+          competition from 8 June to 7 July.
+        </p>
+        <img src="./images/attribution.png" alt="F1 Sustainability Logo" />
```
- [ ] OPTIONAL: The content of this section overlaps with other content on tablet screens, I strongly recommend to address this issue: 
<img width="602" alt="Screenshot 2022-03-10 at 11 57 38" src="https://user-images.githubusercontent.com/77038610/157614877-79cae4e5-fe42-4e7f-8ae1-59d2e5fbe867.png">

Benmuiruri replied:
Thanks, I will address that issue. I noticed it as well.

ShoiraTa commented on README.md:
```diff
@@ -0,0 +1,52 @@
+<h1 align="center"> Module One Capstone Project </h1>
+
+> This is a repository for my Module One Capstone Project at Microverse. It is a website for a fictitious, upcoming F1 Sustainability Summit.
+> Visit the [Live Link](https://benmuiruri.github.io/f1-sustainability-capstone/) to view the website.
+## Built With
```
- [ ] OPTIONAL:  The live link is broken, it opens an empty page, please make sure that the link works as expected

Benmuiruri replied:
The live link is empty because I am yet to merge. I had not set it to render from this branch. Once I merge the page will render as expected.

ShoiraTa reviewed (approved):
## Status: Approved :heavy_check_mark:

Hi @Benmuiruri ,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

![parks-and-rec-great-job-gif](https://user-images.githubusercontent.com/77038610/154920223-207674a1-8f2c-4be6-9d36-fb870b691e1e.gif)

## Optional suggestions

_Check comments below_

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

***
Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Benmuiruri commented:
Thanks, @ShoiraTa, @prolajumokeoni, and @Donard97 for your input and review of my Capstone project.
